codecs/rotate/benchmark.js

```diff
@@ -0,0 +1,45 @@
+// THIS IS NOT A NODE SCRIPT
+// This is a d8 script. Please install jsvu[1] and install v8.
+// Then run `npm run --silent benchmark`.
```

Comment:
tip: `-s`

Reply:
Are we golfing `npm` invocations now? ^^ (Thanks!)